### ISSUE TITLE
Topic: Licensing

### ISSUE THREAD

**Issue by gitcommitshow** [fossified/podcast#5]
I propose to start with something basic - different types of licenses for open-source, and their challenges/benefits, etc. so an open-source author can make better decisions for their next open-source projects from the start.

**Comment by hesa** [comment]
We did a similar thing over at https://foss-north.se/pod/episodes.html. I guess the focus there (and then) was focused on the user of FOSS components rather your proposal with focus on a developer choosing a license. 

What I think about directly are the following things:
* attracting many users
* attracting companies to use your component
* protecting the user's freedom
and also things like:
* using CLA, DCO
with guests:
* one from a permissively licensed project
* one from a GPL/EPL licensed project

Does this sound reasonable? ....are you thinking more in lines of a flow chart to chose a license?

Do you have any hints on what more to include?

**Comment by gitcommitshow** [comment]
Yes, all of that. And it will be amazing if we can present them with real work anecdotes, examples to showcase the importance and gotchas.
Btw, now that you listed down everything, it looks like more than one episode :)
